src/olympia/reviewers/forms.py

```diff
@@ -360,6 +361,17 @@ def __init__(self, *args, **kw):
             (k, v['label']) for k, v in self.helper.actions.items()
         ]
 
+        # Set the queryset for reasons based on the add-on type.
+        self.fields['reasons'].queryset = ReviewActionReason.objects.filter(
+            is_active__exact=True,
```

Comment:
out of interest, what's the `__exact` for?  (I missed it before)

Reply:
I don't recall. Maybe just me misunderstanding the requirements of Django. I take it from your comment that I could just have used `is_active=True`?

Reply:
Yeah you don't need `__exact` most of the time

Reply:
I tried it and it seems to work. 😄